test/SILOptimizer/Mirror-LoadableByAddress-failure.swift

```diff
@@ -0,0 +1,35 @@
+//===--- Mirror.swift -----------------------------------------------------===//
```

Comment:
Can you please remove the copyright notice? we don't have those on the smoke test suite.